### DISCUSSION TITLE
What happens to non-matched comments?

### DISCUSSION THREAD

**Discussion by QuentinMcCarthy** [ThioJoe/YT-Spammer-Purge#630] [General]
The latest update added this "Non-matched comments" section in the results page; but I'm don't entirely sure what that entails. There's nothing in the program that says what happens to those comments. Are they included in the final action (i.e. reported)? Or are they ignored? And if it's the latter, how do you include them?

**Comment by ThioJoe** [comment]
Yes they get the same treatment as all the other comments, for whatever you choose. Perhaps I'll add something that makes that clearer. I see it does still say something like 'how do you want to handle the matched comments above', so I'll change it to say 'ALL' comments above.
